Sources/Sentry/SentrySwizzle.m

```diff
@@ -102,11 +104,59 @@ @implementation SentrySwizzle
 
     pthread_mutex_lock(&gLock);
 
+#if TEST
+    IMP currImp = class_getMethodImplementation(classToSwizzle, selector);
```

Comment:
`m`: Can't we use the `originalIMP` instead of this?

Reply:
No. `originalIMP` may be null if the class don't have a direct implementation of it (inheritance).